src/dd-win-prof/ProfileExporter.cpp

```diff
@@ -364,6 +390,67 @@ std::optional<ddog_prof_LocationId> ProfileExporter::InternLocation(uint64_t add
 }
 
 
+std::optional<ddog_prof_MappingId> ProfileExporter::InternMapping(const CachedSymbolInfo& symbolInfo, ddog_prof_Profile* profile)
+{
+    // Create a cache key based on module name and build ID
+    // Use a simple hash combination of the two IDs for uniqueness
+    uint64_t mappingKey = (static_cast<uint64_t>(symbolInfo.ModuleNameId.value) << 32) |
```

Comment:
💭 I do not think we needed to combine these as we could simply have a hash function that takes both of these.